Resources/Prototypes/_RMC14/Entities/Objects/Devices/command_tablet.yml

```
@@ -0,0 +1,18 @@
+- type: entity
+  parent: BaseItem
+  id: RMCTabletCO
+  name: command tablet
+  description: A portable command interface used by top brass, capable of issuing commands over long ranges to their linked computer. Built to withstand a nuclear bomb.
+  components:
+  - type: Sprite
+    sprite: _RMC14/Objects/Devices/command_tablet.rsi
+    state: cotablet
+  - type: Item
+    size: Normal
+  - type: ActivatableUI
```

```suggestion
  - type: ActivatableUI
  - type: ActivatableUIRequiresAccess
  - type: AccessReader
    access: [ [ "CMAccessCommand" ] ]
```
This should only be usable by command access